.github/workflows/examples.yml

```
@@ -33,7 +33,7 @@ jobs:
     - name: Install (apt)
       run: |
         sudo apt-get update
-        sudo apt-get -y install openmpi-bin libopenmpi-dev
+        sudo apt-get -y install openmpi-bin libopenmpi-dev h
```

Is this `h` intentional?
```suggestion
        sudo apt-get -y install openmpi-bin libopenmpi-dev
```

---

not, I don't know where this h comes from and what this h means.